docs/site/src/demos/tabs/tabs.md

```diff
@@ -31,3 +31,9 @@ Tab labels may be either all icons or all text.
 
 {{demo='demos/tabs/IconTabs.js'}}
 {{demo='demos/tabs/IconLabelTabs.js'}}
+
+## Disabled Tab
```

Comment:
I am not sure which example to modify. There are four here

1. Basic Tabs
2. Full width Tabs
3. Centered
4. Icon Tabs

Can you point out which should be modified?